test/rest.test.js

```diff
@@ -55,6 +55,8 @@ describe('strong-remoting-rest', function() {
 
     // connect to the app
     objects.connect('http://localhost:' + server.address().port, adapterName);
+    var optsErrorHandler = { errorHandler: { debug: true, log: false }};
+    extend(objects.options, optsErrorHandler);
```

Comment:
Instead of extending `objects.options`, please modify the line 53 where the default errorHandler options are created.

This is what we have right now:

```js
objects = RemoteObjects.create({ json: { limit: '1kb' },
       errorHandler: { disableStackTrace: false }});
```

I believe we no longer support `disableStackTrace`, in which case it should be removed from tests too.